@navikt/ds-css/accordion-menu.css

```diff
@@ -1,3 +1,10 @@
+:root {
+  --navds-accordion-menu-color: var(--navds-color-text-link);
```

Comment:
Jeg tror det er lurt å kalle denne for `--navds-accordion-menu-color-text` selv om css-propen bare heter color.

Reply:
Bestemte at vi prøver med `-text` for nå, så får vi se om det fungerer